gcc/rust/expand/rust-macro-expand.h

```diff
@@ -193,6 +193,28 @@ struct MacroExpander
   bool match_matcher (Parser<MacroInvocLexer> &parser,
 		      AST::MacroMatcher &matcher);
 
+  /**
+   * Match any amount of matches
```

Comment:
I believe it is but haven't checked :D This is just how I write doc since I believe it's also a nice formatting when looking directly at the header @dkm what do you think?

Reply:
Not really sure, doxygen supports many idioms. But do we want doxygen ? I thought that as nobody replied on Zulipp, I was simply the only one using it because my emacs setup lacks some love. I would really love having the same kind of doc as `rustc` has for its internal and doxygen is the only tool I know for C++.

Reply:
I personally really like this style of header documentation, but find Doxygen itself to be an absolute hassle. It's annoying to set up or open, or maybe it's my fault for not taking the time to understand it. Which is why I was unsure about what to say on Zulip. I believe the gcc project is using/switching to Sphinx, which looks interesting but I've never used it.

Reply:
GCC is moving to sphinx for the general doc, its content is not extracted from the source... I don't really like doxygen but don't know anything else :)